DATA/production/configurations/asyncReco/async_pass.sh

```diff
@@ -620,8 +625,8 @@ else
       echo "WORKFLOW_PARAMETERS=$WORKFLOW_PARAMETERS"
       echo "Step 4) QC"
       echo -e "\nStep 4) QC" >> workflowconfig.log
-      export TIMEFRAME_RATE_LIMIT=${ALIEN_JDL_TIMEFRAMERATELIMITSSPLITWF:-0}
       echo "Removing detectors $DETECTORS_EXCLUDE"
+      echo "The rate limiting will be the same as in step 3: TIMEFRAME_RATE_LIMIT = ${TIMEFRAME_RATE_LIMIT}"
```

Comment:
Perhaps one should do 
```
      export TIMEFRAME_RATE_LIMIT=${ALIEN_JDL_TIMEFRAMERATELIMITSSPLITWF:-${TIMEFRAME_RATE_LIMIT}}
```
so that by default the same limit is used but we have a possibility to override it? Most probably, the same limit will work, just in case.

Reply:
This is the 4th step for QC, where again we use the reader. So I think it should be configured as the step above. If we add what you suggest, it will be like the previous step. If you want to differentiate it, we need another JDL variable. 
Since all this is not yet finalized, maybe we could for not leave it assuming step 3 and step 4 are to be configured in the same way?

Reply:
Fine for me, we can add special setting later, if needed.